src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -460,6 +470,12 @@ public void UpdateEnergyBalance(EnergyBalanceInfo energyBalance)
             atpBalanceLabel.AddColorOverride("font_color", new Color(1.0f, 0.2f, 0.2f));
         }
 
+        atpProductionLabel.Text = $"{energyBalance.TotalProduction}";
```

Comment:
Shoud be formatted to as how it was before: `string.Format(CultureInfo.CurrentCulture, "{0:F1}", energyBalance.TotalProduction);` (and the same for `TotalConsumption`) to limit the decimals to only one digit